src/Writers/HTMLWriter.jl

```diff
@@ -490,7 +491,7 @@ mutable struct SearchIndexBuffer
         page_title = mdflatten(pagetitle(ctx, navnode))
         new(
             ctx,
-            pretty_url(ctx, get_url(ctx, get(navnode.page))),
+            pretty_url(ctx, get_url(ctx, navnode.page)),
```

Comment:
This one is a bit more complex since `get_url` accepts any type for the second argument. But specifying `::AbstractString` sounds like a good idea anyway, which I just added the commit.